core/local/watcher.js

```diff
@@ -20,20 +19,27 @@ export interface Watcher {
 }
 */
 
-const log = logger({
-  component: 'LocalWatcher'
-})
-
 function build (syncPath /*: string */, prep /*: Prep */, pouch /*: Pouch */, events /*: EventEmitter */, ignore /*: Ignore */) /*: Watcher */ {
+  let watcher = 'atom'
+  if (process.platform === 'darwin') {
+    watcher = 'chokidar'
+  }
   const env = process.env.COZY_FS_WATCHER
   if (['experimental', 'atom'].includes(env)) {
-    try {
-      return new AtomWatcher(syncPath, prep, pouch, events, ignore)
-    } catch (err) {
-      log.error({err}, 'Cannot use AtomWatcher')
-    }
+    watcher = 'atom'
+  } else if (env === 'chokidar') {
+    watcher = 'chokidar'
+  }
+  // FIXME Integration and scenario tests use ChokidarEvents
+  // and are not yet compatible with atom/watcher
+  if (process.env) {
```

Comment:
I'm not sure to understand this condition :upside_down_face:

Reply:
We are using `onFlush` for those tests, and the `onFlush` method is defined only on the old watcher, not the new watcher. And those tests create some `ChokidarEvents` (with a doctype, not a kind).